app/controllers/account/admin/issues_controller.rb

```diff
@@ -37,13 +40,28 @@ def destroy
 
       private
 
+      def facebook_posting(issue)
+        return if Rails.env.test?
+        page = prepare_facebook_page
+        page.feed!(message: "Адреса: #{issue.location}. \n \n
```

Comment:
description can't be blank, we have validation for it) also tags added to all categories

but anyway how to refactor it? Check if tags or description is blank and add new lines only if they're not blank?